diff --git a/node_modules/react-native-fbads/android/app/build.gradle b/node_modules/react-native-fbads/android/app/build.gradle
index 39e889c..1fc20c6 100644
--- a/node_modules/react-native-fbads/android/app/build.gradle
+++ b/node_modules/react-native-fbads/android/app/build.gradle
@@ -5,12 +5,12 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 26)
-    buildToolsVersion safeExtGet('buildToolsVersion', '26.0.3')
+    compileSdkVersion 34
+    buildToolsVersion '34.0.0'
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 26)
+        minSdkVersion 21
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
         ndk {
@@ -26,7 +26,7 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation "com.android.support:recyclerview-v7:${safeExtGet('supportLibVersion', '26.1.0')}"
-    implementation 'com.facebook.android:audience-network-sdk:6.2.+'
-    implementation 'com.facebook.android:facebook-android-sdk:6.+'
+    implementation "com.android.support:recyclerview-v7:26.1.0"
+    implementation 'com.facebook.android:facebook-android-sdk:16.3.0'
+    implementation 'com.facebook.android:audience-network-sdk:6.16.0'
 }
